content/docs/usage/certificate.md

```diff
@@ -256,6 +256,44 @@ data:
   ...
 ```
 
+### Creating Certificate With Name Constraints
+
+Root or Intermediate CA certificates can have NameConstraints. Name constraints indicates a name space within which all subject names in subsequent certificates in a certification path MUST be located.
+Checkout https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.10 for more details on this.
+
+This feature is only enabled by adding it to the
+`--feature-gates` flag on the cert-manager controller and webhook components:
+
+```bash
+--feature-gates=useCertificateRequestNameConstraints=true
+```
```

Comment:
**Suggestion:** warn the user that this feature is an alpha feature.

~~~suggestion
<div className="warning">

⛔️ The name constraints feature currently in an alpha state, and is subject to breaking changes or complete removal in future releases. This feature is only enabled by adding it to the `--feature-gates` flag on the cert-manager controller and webhook components:

```bash
--feature-gates= useCertificateRequestNameConstraints=true
```

</div>
~~~